runtime/compiler/control/rossa.cpp

```diff
@@ -1374,6 +1374,16 @@ onLoadInternal(
 
    TR::CodeCacheConfig &codeCacheConfig = codeCacheManager->codeCacheConfig();
 
+   if (TR::Options::_overrideCodecachetotal && TR::Options::isAnyVerboseOptionSet(TR_VerbosePerformance,TR_VerboseCodeCache))
+      {
+      // Code cache total value defaults were overridden due to low memory available
+      bool incomplete;
+      uint64_t freePhysicalMemoryB = getCompilationInfo(jitConfig)->computeFreePhysicalMemory(incomplete);
+      if (freePhysicalMemoryB != OMRPORT_MEMINFO_NOT_AVAILABLE)
+         TR_VerboseLog::writeLineLocked(TR_Vlog_CODECACHE, "Available freePhysicalMemory=%llu MB, allocating code cache total size=%llu MB", freePhysicalMemoryB >> 20, jitConfig->codeCacheTotalKB >> 10);
```

Comment:
Please make sure that we have the correct printf specifiers. jitConfig->codeCacheTotalKB is of type `UDATA` which is 32 bit on 32-bit platforms. `%llu` is for unsigned long long which is typically 64 bit even on 32-bit platforms